frontend/src/app/App.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import '@patternfly/react-core/dist/styles/base.css';
+import '@patternfly/patternfly/patternfly.min.css';
```

Comment:
Don't we need the react-core css? I thought pulling in the base.css pulled in patternfly css.

Reply:
the base styles pull in the variables mostly, and then the react components import the individual styles, like alert.css etc. This is why in dev mode you see a lot of <style> tags in the header.
The other approach is to just pull in the patternfly.css, and we can use the webpack null-loader to disable style injection of the individual components.
Without this change the quick start drawers did not look correct, I am not exactly sure why but i think it's because the component CSS imports were overriding styles.

Reply:
OK for now. I'd like to get to the bottom of this at some point though. We should only need to import CSS in one place and not need to know about all the css files that dependencies require.

Reply:
Ah i remember now why, the quickstarts.css doesn't include the PatternFly styles that it depends on, simply because I didn't want those styles to be double loaded in apps that already use PatternFly.
So if the drawer uses the class `.pf-c-wizard__nav-link` for instance, the app doesn't know about it because we are not loading the Wizard.css stylesheet.
That's why when this is changed to import the whole patternfly.css stylesheet, these are all understood.